src/controllers/room.controller.js

```diff
@@ -0,0 +1,81 @@
+import { Room } from '../models/Room.models.js';
+
+export const getRooms = async (req, res) => {
+  res.send(await Room.findAll());
+};
+
+export const getOneRoom = async (req, res) => {
+  const { id } = req.params;
+  const room = await Room.findByPk(id);
+
+  if (!room) {
+    res.status(404);
+
+    return;
```

Comment:
When an error occurs, it is a good practice to send some information about the error to the client. In this case, after setting the status to 404, you should send a message to the client informing that the room was not found.